ui/components/app/transaction-activity-empty-state/transaction-activity-empty-state.test.tsx

```diff
@@ -58,13 +176,104 @@ describe('TransactionActivityEmptyState', () => {
 
   it('renders dark theme image when dark theme is selected', () => {
     renderComponent(
-      {},
-      { metamask: { ...mockState.metamask, theme: ThemeType.dark } },
+      {}, // no prop changes
+      {
+        metamask: { ...mockState.metamask, theme: ThemeType.dark },
+      },
     );
     const image = screen.getByAltText('Activity');
     expect(image).toHaveAttribute(
       'src',
       './images/empty-state-activity-dark.png',
     );
   });
+
+  describe('Swap button functionality', () => {
+    const accountWithoutSigning = createAccount({
+      methods: ['personal_sign'], // No eth_signTransaction or eth_signUserOperation
+    });
+
+    const accountWithSigning = createAccount({
+      methods: [EthMethod.SignTransaction, 'personal_sign'],
+    });
+
+    // TODO: Our jest describe is typed as Mocha this should be fixed
+    (describe as unknown as jest.Describe).each<
```

Comment:
For some reason our jest describe is typed as Mocha and not Jest??

<img width="580" height="148" alt="Image" src="https://github.com/user-attachments/assets/1dd5b0e3-6065-4479-b601-7fc604f23b4c" />

We should be able to do a [describe.each](https://jestjs.io/docs/api#describeeachtablename-fn-timeout). If you search the code base this type error is disabled. If you search the codebase they're all disabled